src/Coordination/FourLetterCommand.h

```diff
@@ -390,4 +389,16 @@ struct RecalculateCommand : public IFourLetterCommand
     ~RecalculateCommand() override = default;
 };
 
+struct CleanResourcesCommand : public IFourLetterCommand
+{
+    explicit CleanResourcesCommand(KeeperDispatcher & keeper_dispatcher_)
+        : IFourLetterCommand(keeper_dispatcher_)
+    {
+    }
+
+    String name() override { return "clrs"; }
```

Comment:
Fun fact: the abbreviation `CLRS` is also the short name of the "Introduction to Algorithms" book.

Probably because you may want to clear some resources of your mind before soaking up the knowledge it offers :)